docs/snippets/intro_snippet_test.py

```diff
@@ -0,0 +1,7 @@
+
+from tests.pipeline.utils import assert_load_info
+from docs.snippets.utils import run_snippet
```

Comment:
add fixtures: preserve_environ, autouse_test_storage, patch_home_dir so the tests are isolated